src/mrinufft/trajectories/utils.py

```diff
@@ -1,47 +1,156 @@
 """Utility functions for the trajectory design."""
-import warnings
 import numpy as np
 
+
 #############
 # CONSTANTS #
 #############
 
 KMAX = 0.5
 
-DEFAULT_RESOLUTION = 6e-4  # 0.6 mm isotropic
 DEFAULT_CONE_ANGLE = np.pi / 2  # rad
 DEFAULT_HELIX_ANGLE = np.pi  # rad
 
-DEFAULT_RASTER_TIME_MS = 10e-3  # ms
-DEFAULT_GYROMAGNETIC_RATIO = 42.576e3  # MHz/T
+CARBON_GYROMAGNETIC_RATIO = 10708.4  # kHz/T
+HYDROGEN_GYROMAGNETIC_RATIO = 42576.384  # kHz/T
+PHOSPHOROUS_GYROMAGNETIC_RATIO = 17235  # kHz/T
+SODIUM_GYROMAGNETIC_RATIO = 11262  # kHz/T
```

Comment:
I would suggest the use of a Python Enum for all those constant , and then use `Gyro.Na` or `Gyro.Sodium` (or both, Enum accepts aliases)

Reply:
An enum has been implemented with additional values and aliases